api/src/integration/java/org/pmiops/workbench/FireCloudIntegrationTest.java

```diff
@@ -41,17 +50,18 @@
         Providers.of(workspacesApi),
         new FirecloudRetryHandler(new NoBackOffPolicy())
     );
+  }
+
+  @Test
+  public void testStatus() {
+    assertThat(fireCloudService.getFirecloudStatus()).isTrue();
+  }
 
-    @Test
-    public void testStatus() {
-        assertThat(fireCloudService.getFirecloudStatus()).isTrue();
-    }
-
-    private static WorkbenchConfig createConfig() {
-        WorkbenchConfig config = new WorkbenchConfig();
-        config.firecloud = new WorkbenchConfig.FireCloudConfig();
-        config.firecloud.debugEndpoints = true;
-        return config;
-    }
+  private static WorkbenchConfig createConfig() throws Exception {
```

Comment:
How does using config_local.json affect the functioning of this test (if at all?)

(It doesn't necessarily hurt anything... I'm just curious what prompted the change here.)

Reply:
The only reason to use the config is to get the base firecloud url that the status call hits. I didn't want to hard-code that into the test itself

Reply:
It was an intentional change ... but I'm rethinking it now. This is an integration test so maybe this should be the production url. Yeah ... the more I think about it, the more I lean towards hitting prod, not dev.

Reply:
I was having trouble with that. For autowired, the app code needs a config to do the population of the client. The test needs to also provide a config, any config, but I could not get the autowired to work so resorted to a manual status api creation.

Reply:
So previously, createConfig() just constructed a WorkbenchConfig that didn't read in any JSON file... the only thing that got populated was debugEndpoints. I was wondering why that didn't continue working the way it was.

There is a way of doing all this with @Autowired too, FWIW, but I don't know that it's worth bothering in this case.

Anyway, this is fine, too.

Reply:
The previous way of creating a config didn't work due to not having a base url. Since we use that in the construction in `FirecloudConfig` we need a real value to populate it with.